courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<p>From <a href="http://pep8online.com/" target="_blank">http://pep8online.com</a></p><p><strong>E261 </strong>at least two spaces before inline comment</p><p><strong>E262 </strong>inline comment should start with '# '</p><p> </p><p>This cell feels confusing, since it's not clear how to use this function for training.</p><p>If we keep it, we should have also the code to be able to train it (training loop from scratch with</p><p>tf.GrandientTape to compute gradients?). Personally, I'd remove it altogether, otherwise the </p><p>training code can be adapted from: <a href="https://github.com/GoogleCloudPlatform/training-data-analyst/blob/ml_on_gcp/courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/training-data-analyst/blob/ml_on_gcp/courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Yeah, it's a carry over from the older lab, so I'm not surprised it seems out of place. I'm not confident it adds much value either. I'll go ahead and remove it.</p>